documentation/release_notes.rst

```diff
@@ -20,7 +20,7 @@ New Features
   with device policies for large data sizes.
 - Improved performance of ``copy``, ``fill``, ``for_each``, ``replace``, ``reverse``, ``rotate``, ``transform`` and 30+
   other algorithms with device policies on GPUs.
-- Improved oneDPL use with SYCL implementations other than Intel oneDPI DPC++/C++ compiler.
+- Improved oneDPL use with SYCL implementations other than Intel® oneDPI DPC++/C++ compiler.
```

Comment:
- oneDPI needs to change to oneAPI 
- compiler needs to be capitalized "Compiler"

Reply:
I've made the changes.  Please confirm and approve the PR.  Thank you!

Reply:
I am still seeing a lowercase c in compiler, once that is uppercase, we can close this PR.

Reply:
I've got that fixed now.  Thank you.